### DISCUSSION TITLE
Testnet

### DISCUSSION THREAD

**Discussion by abatakunn** [pkgxdev/pkgx#954] [Help] [answered]
Testnet is Live now?

**Comment by tandotendo** [comment, marked as answer]
Hello friends, currently the testnet hasn't started yet. The info is that it will start on Feb 21. For further information, you can open the following link https://tea.xyz/

**Comment by minhdataaa** [comment]
Let's join on Feb-21 !

**Comment by hanriiver** [comment]
Testnet is Feb-21
mainnet is June

You get more informations go to official twitter
https://twitter.com/teaprotocol

**Comment by Oknessman** [comment]
Pas encore mais dans exactement 22jours
